src/main/java/ru/hse/sd/cli/exceptions/ParserException.java

```diff
@@ -0,0 +1,7 @@
+package ru.hse.sd.cli.exceptions;
+
+public class ParserException extends Exception{
```

Comment:
```suggestion
public class ParserException extends Exception {
```
И исключениям тоже нужны комментарии